setup.sh

```diff
@@ -5,6 +5,6 @@
 git clone git@github.com:makerdao/dss-deploy-scripts.git src/deployment-scripts
 pushd src/deployment-scripts
 git fetch --all --tags --prune
-git checkout tags/1.0.5
+git checkout master
```

Comment:
Is this going to be update to last version when will dar?

Reply:
@gbalabasquer can answer this.  At the moment, I saw no branch/tag for it.